File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/mix/MixAccountApi.java
package com.bitget.openapi.api.mix;

import com.bitget.openapi.dto.request.mix.*;
import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.Query;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Contract account api
 */
public interface MixAccountApi {

    /**
     * Get account  information
     * @param symbol
     * @param marginCoin
     * @return ResponseResult
     */
    @GET("/api/mix/v1/account/account")
    Call<ResponseResult> account(@Query("symbol") String symbol,
                                 @Query("marginCoin") String marginCoin);
    /**
     * Get account  information list
     * @param productType
     * @return ResponseResult
     */
    @GET("/api/mix/v1/account/accounts")
    Call<ResponseResult> accounts(@Query("productType") String productType);

    /**
     * set lever
     * @param mixChangeLeverageReq
     * @return ResponseResult
     */
    @POST("/api/mix/v1/account/setLeverage")
    Call<ResponseResult> leverage(@Body MixChangeLeverageReq mixChangeLeverageReq);

    /**
     * Adjustment margin
     * @param mixAdjustMarginFixReq
     * @return ResponseResult
     */
    @POST("/api/mix/v1/account/setMargin")
    Call<ResponseResult> margin(@Body MixAdjustMarginFixReq mixAdjustMarginFixReq);

    /**
     * Adjust margin mode
     * @param adjustMarginModeReq
     * @return ResponseResult
     */
    @POST("/api/mix/v1/account/setMarginMode")
    Call<ResponseResult> marginMode(@Body AdjustMarginModeReq adjustMarginModeReq);

    /**
     * Adjust hold mode
     * @param adjustHoldModeReq
     * @return ResponseResult
     */
    @POST("/api/mix/v1/account/setPositionMode")
    Call<ResponseResult> positionMode(@Body AdjustHoldModeReq adjustHoldModeReq);

    /**
     * Get the openable quantity
     * @param mixOpenCountReq
     * @return ResponseResult
     */
    @POST("/api/mix/v1/account/open-count")
    Call<ResponseResult> openCount(@Body MixOpenCountReq mixOpenCountReq);
}

File: bitget-php-sdk-api/src/api/mix/MixAccountApi.php
<?php


namespace bitget\api\mix;


use bitget\internal\BitgetApiClient;
use bitget\model\mix\account\OpenCountReq;
use bitget\model\mix\account\SetLeverageReq;
use bitget\model\mix\account\SetMarginModeReq;
use bitget\model\mix\account\SetMarginReq;
use bitget\model\mix\account\SetPositionModeReq;

class MixAccountApi
{
    const BASE_URL = "/api/mix/v1/account";

    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    /**
     * Get account  information
     * @param  $symbol
     * @param $marginCoin
     * @return string
     */
    public function account(string $symbol, string $marginCoin): string
    {
        $params = array("symbol" => $symbol, "marginCoin" => $marginCoin);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/account", $params);
    }
    /**
     * Get account  information list
     * @param $productType
     * @return string
     */
    public function accounts(string $productType): string
    {
        $params = array("productType" => $productType);

        return $this->BitgetApiClient->doGet(self::BASE_URL . "/accounts", $params);
    }
    /**
     * set lever
     * @param $setLeverageReq
     * @return string
     */
    public function setLeverage(SetLeverageReq $setLeverageReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/setLeverage", $setLeverageReq);
    }
    /**
     * Adjustment margin
     * @param $setMarginReq
     * @return string
     */
    public function setMargin(SetMarginReq $setMarginReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/setMargin", $setMarginReq);
    }
    /**
     * Adjust margin mode
     * @param $setMarginModeReq
     * @return string
     */
    public function setMarginMode(SetMarginModeReq $setMarginModeReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/setMarginMode", $setMarginModeReq);
    }
    /**
     * Adjust hold mode
     * @param $setPositionModeReq
     * @return string
     */
    public function setPositionMode(SetPositionModeReq $setPositionModeReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/setPositionMode", $setPositionModeReq);
    }
    /**
     * Get the openable quantity
     * @param $openCountReq
     * @return string
     */
    public function openCount(OpenCountReq $openCountReq)
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/open-count", $openCountReq);
    }


}